src/App.vue

```diff
@@ -93,6 +93,15 @@ li {
   flex-direction: column;
   position: relative;
 }
+.float-button {
+  position: fixed;
```

Comment:
Zrobiłem sobie trochę testów, m.in na mobilkach i niestety przycisk się rozjeżdza (ucieka do góry). Wrzucę screeny na slacka do oglądu.

Myślę że powinniśmy raczej skorzystać z tego że element `#app` jest `position:relative` i w związku z tym możemy użyć `position:absolute` + `bottom:` który będzie zamiast pozycjonować się względem ekranu to będzie robił to względem elementu `#app`. Już niektóre elementy tak pozycjonujemy, np. w komponencie `Quest`.

Reply:
Position absolute, a sticky element to zupełnie różne rzeczy, więc to się nie zda

Reply:
> Position absolute, a sticky element to zupełnie różne rzeczy, więc to się nie zda

Możesz wytłumaczyć co masz dokładnie na myśli? Aktualnie już używamy `position: absolute` do pozycjonowania względem "ramki" aplikacji (`#app` i IMO można w ten sam sposób zrobić ten przycisk. Akurat `position: fixed` wydaje się tu być mniej sensowne jako że jest pozycjonowane zawsze względem ekranu (ale tu się nie znam).

@fraunos może masz jakąś opinię na ten temat?

Reply:
To, że jest pozycjonowane zawsze względem ekranu, to wypadkowa tego że fixed realizuje funkcjonalność sticky. position absolute znika jak scrollujesz ekran

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/position

Reply:
> position absolute znika jak scrollujesz ekran

Nie do końca rozumiem co miałeś tu na myśli. Czy chodzi o to, że jeśli jest overflow w kontenerze to przycisk będzie się przesuwał razem z zawartością przy scrollowaniu?

Jeśli tak, to możesz przecież mieć:
1. przycisk pozycjonowany absolutnie względem elementu `#app`
2. dodatkowy `<div>` w `#app` na w którym będzie się scrollował właściwy content (a nie w samym `#app`) - być może będzie konieczne ustawienie odpowiednio `overflow`. Sam `#app` będzie miał tę samą wysokość (przy założeniu, że nie skalujemy ekranu ofc).

I nie powinieneś mieć wtedy tego problemu o którym mówisz.

Mogę wystawić PR żeby się upewnić że to rozwiązanie działa.

Reply:
@apragacz  @ppeczek  hejka, próbuję to zrobić, to zaraz może też otworzę PR

Reply:
https://github.com/fajnie-ze-wiesz/fzw-webapp/pull/73
no ja miałem obawy co do tego guzika, że nie będzie się dało tego zrobić idealnie, tylko będzie jakieś druciarstwo no i mi też trochę krzywo wyszło, ale zobaczcie co sami sądzicie
@apragacz  @ppeczek